dropbox/dropbox.py

```diff
@@ -149,6 +193,9 @@ def __init__(self,
             server. After the timeout the client will give up on
             connection. If `None`, client will wait forever. Defaults
             to 30 seconds.
+        :param PathRoot path_root: the path root object to serialize and use for
+            the ``Dropbox-API-Path-Root`` header. (If ``None``, the header is
+            cleared, which restores the default behavior.)
```

Comment:
There's documentation for `path_root` as an argument, but it's not actually included in the list of args for `__init__`

Reply:
This was intentional, but perhaps needs to be rethought.

`__init__` does not take a `path_root` directly. It takes `headers`. Rather than try to reconcile any competing header value, the intention was to allow crafting of custom headers using the `clear_path_root_header` and `set_path_root_header` helper functions before calling `__init__`. E.G.:

```
# Uses intermediate object:
dbx_without_path_root = Dropbox(…)
dbx_with_path_root = dbx_without_path_root.with_path_root(PathRoot(…))

# Avoids intermediate object:
headers = {}
set_path_root_header(headers, PathRoot(…))
dbx_with_path_root = Dropbox(…, headers=headers)
```